libavfilter/dnn/dnn_backend_native_layer_mathbinary.c

```diff
@@ -102,95 +160,23 @@ int dnn_execute_layer_math_binary(DnnOperand *operands, const int32_t *input_ope
     dst = output->data;
 
     switch (params->bin_op) {
-    case DMBO_SUB:
-        if (params->input0_broadcast) {
-            for (int i = 0; i < dims_count; ++i) {
-                dst[i] = params->v - src[i];
-            }
-        } else if (params->input1_broadcast) {
-            for (int i = 0; i < dims_count; ++i) {
-                dst[i] = src[i] - params->v;
-            }
-        } else {
-            const DnnOperand *input1 = &operands[input_operand_indexes[1]];
-            const float *src1 = input1->data;
-            for (int i = 0; i < dims_count; ++i) {
-                dst[i] = src[i] - src1[i];
-            }
-        }
+    case DMBO_SUB:  
+        mathbin2(SUB_fun, params, src, dst, dims_count, operands, input_operand_indexes); 
```

Comment:
if we pass input and output here,
the dims_count is not needed, right?

Reply:
This suggestion has been adopted in the new patch.